taskbench.c

```diff
@@ -43,10 +43,11 @@ int main(int argc, char **argv) {
 
     init(argc, argv);
 
-#ifdef OMPVER3
+//#ifdef OMPVER3
 
     /* GENERATE REFERENCE TIME */
     reference("reference time 1", &refer);
+	//printf("\t%d default inner iterations\n",innerreps);
```

Comment:
Does this need to be commented out or removed?

Reply:
Yes removing